hack/init_build.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -u
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+. "${ROOT}/hack/utilities.sh" || { echo 'Cannot load bash utilities.'; exit 1; }
+
+VERSIONS="/tmp/.versions"
+GO_VERSION='1.7.3'
```

Comment:
I believe we have other scripts that define some of these values too - perhaps we should put them all in a common file to avoid them getting out of sync?